pyproject.toml

```diff
@@ -78,6 +85,8 @@ mdformat = "^0.7.14"
 mdformat_myst = "^0.1.4"
 mdformat-toc = "^0.3.0"
 pip-audit = "^2.1.0"
+types-requests = "^2.32.0"
+requests="^2.32.1"
```

Comment:
`requests` was installed by `type-requests`

Reply:
We need requests for dev scripts. @jfrery had removed it some time ago but by keeping `type-requests` we were in fact still installing it. It was weird to me that we had the type stubs of a packages that we weren't requiring. So I tried removing it but everything broke. So I added back requests here since I feel like it's more explicit than relying on the fact that `type-requests` requires `requests`.

Reply:
hmm interesting, gotcha